core/backend/src/IModelDb.ts

```diff
@@ -1229,6 +1243,7 @@ export abstract class IModelDb extends IModel {
    */
   public querySchemaVersion(schemaName: string): string | undefined {
     const sql = `SELECT VersionMajor,VersionWrite,VersionMinor FROM ECDbMeta.ECSchemaDef WHERE Name=:schemaName LIMIT 1`;
+    // eslint-disable-next-line @typescript-eslint/no-deprecated
```

Comment:
I believe we need to deprecate this method as well. We cannot rewrite it to async without breaking its signature. Once we remove statement this will have to become async